### PR TITLE
[TACHYON 1021] Write JavaDoc for tachyon.master.file.journal.RenameEntry

### PR DESCRIPTION
**Pull request by saveveltri**
https://tachyon.atlassian.net/browse/TACHYON-1021

Hi everyone,
This is my second NewContributor Issue and my first comments related issue.
Feel free to post in case of any mistakes or misunderstandings

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

File: servers/src/main/java/tachyon/master/file/journal/RenameEntry.java [hunk 1]
```diff
@@ -22,11 +22,21 @@
 import tachyon.master.journal.JournalEntry;
 import tachyon.master.journal.JournalEntryType;
 
+/**
+ * This class represents a journal entry for renaming an existing entry.
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/file/journal/RenameEntry.java]: Could you change this to say "renaming a file"? Thanks.

**Comment by gpang** [comment]
Thanks for the PR! I just had a minor suggestion.

**Comment by saveveltri** [comment]
Done. Thanks for the suggestion

**Comment by haoyuan** [comment]
ok to test

**Comment by gpang** [comment]
LGTM

**Pull request merged**